index.html

```diff
@@ -1,18 +1,62 @@
-<script>
-let name = document.location.pathname.slice(1);
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Yelpington</title>
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+    />
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+    <style>
+      #map {
+        height: 400px;
+      }
+      #head {
+        background: green;
+        border-radius: 5px;
+      }
+      body {
+        background: rgb(91, 201, 91);
+      }
+    </style>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div id="header">Yelpington</div>
+    <div id="map"></div>
+    <script>
+      var map = L.map("map").setView([44.477, -73.2144], 17);
```

Comment:
As I've stated on line 13, you should use an external JavaScript file, rather than writing JavaScript inline with <script> tags. 

I've also noticed this file contains multiple script elements all over the place, which is also making this file less organized. This problem will also be solved with the addition of an external JavaScript file.